drafts/draft-lcurley-compressed-mp4.md

```
@@ -0,0 +1,334 @@
+      flags=0x200                        2 bytes
+      sample_size=200                    2 bytes
+mdat (id=1, size=200)                    2 bytes
+  <200 bytes of media data>
+~~~
+
+**Total overhead: ~21 bytes** (excluding media data).
+
+This represents a **78% reduction** in per-fragment overhead (from 96 bytes to ~21 bytes).
+
+
+# Security Considerations
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)